packages/bisect-summary/bisect-summary.0.3/opam

```diff
@@ -0,0 +1,24 @@
+opam-version   : "1.2"
+name           : "bisect-summary"
+version        : "0.3"
```

Comment:
Please don't include the `name` and `version` fields in the opam-repository metadata. `opam-publish` will remove these fields for you automatically to make a pinnable `opam` and a publishable `opam` easier to maintain (we know this is annoying).